features/support/examples/rspec/spec_helper_only_rack_test.rb.example

```diff
@@ -3,7 +3,6 @@ require "rspec/core"
 require "rspec/matchers"
 require "rspec/expectations"
 require "raml-rb"
-require "rack/test"
 
 path = File.expand_path('../../lib', __FILE__)
 
```

Comment:
I'm wondering if we should just delete this file and use only a generic spec helper in case there isn't one already. For that matter, we could just assume there is one already, since Rails apps will have one by default. WDYT?